conda.recipe/meta.yaml

```diff
@@ -69,6 +63,9 @@ test:
   imports:
     - conda
     - conda_env
+  files:
+    - test_conda.bat                # [win]
+    - test_conda.sh                 # [unix]
```

Comment:
Odd, does that mean the test files weren't part of the conda package?

Reply:
These are meant to be smoke tests run after the `conda` conda package is generated; I'll go ahead and remove these references since since (1) the smoke tests are already done in `run_test.*`, and (2) the files are not actually present in the recipe directory (though I'm not sure why conda-build doesn't fail because of that).